frontend/packages/dev-console/src/components/import/advanced/ProgressiveList.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import FormSectionDivider from '../section/FormSectionDivider';
+import './ProgressiveList.scss';
+
+const getGeneratedText = (items: string[], callback) => {
+  if (items && items.length <= 0) {
```

Comment:
Doesn't look like items will ever be `undefined`. However:
```suggestion
  if (!items || items.length === 0) {
```